src/models/base/FirestoreCollection.test.ts

```diff
@@ -1,12 +1,20 @@
-import { clearFirestoreData } from '@firebase/rules-unit-testing';
+import {
+  clearFirestoreData,
+  loadFirestoreRules,
+} from '@firebase/rules-unit-testing';
 import firebase, { firestore } from 'firebase';
+import fs from 'fs';
 import FirestoreCollection from './FirestoreCollection';
 
 const app = firebase.initializeApp({ projectId: 'fs-collection-test' });
 app.firestore().settings({
   host: 'localhost:8080',
   ssl: false,
 });
+loadFirestoreRules({
+  projectId: 'fs-collection-test',
+  rules: fs.readFileSync('./firestore-open.rules', 'utf8'),
+});
```

Comment:
We should refactor this. The last 8 lines here are repetitive across files and are not used elsewhere. Maybe an `initializeTestApp()` function in a `lib/testing.js` file. Then we can replace this with:
```suggestion
const app = initializeTestApp({ projectId: 'fs-collection-test'});
```